release_description.md

```diff
@@ -0,0 +1,29 @@
+The Amazon QLDB team is pleased to announce the release of v1.2.0 of the QLDB .NET driver.
+
+## Enhancements
+* Add the asynchronous version of the driver.
+* Improved retry logic to now handle more types of failure.
+
+The full list of changes are included in the [change log](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/master/CHANGELOG.md).
```

Comment:
Could we also add an additional bullet to highlight bug fixes? e.g., Various bug fixes.

I think this flows better with the next statement directing customers to the change log.

Reply:
Addressed.